irc/config.go

```diff
@@ -352,7 +353,7 @@ type Oper struct {
 	WhoisLine string
 	Vhost     string
 	Pass      []byte
-	Modes     string
+	Modes     []modes.ModeChange
```

Comment:
this makes me nervous. let's do the hack we do elsewhere to make sure we're not gonna run into any trouble with this in the future.
```
ModesString string `yaml:"modes"` 
Modes []modes.ModeChange `yaml:"modes-changes"`
```

Reply:
`Oper` isn't the direct target of `yaml.Unmarshal` (that's `OperConfig`) --- this is already a structure that we transfer "cooked" data into.

Reply:
o. my b. yea in that case cool, all good, thanks a bunch for the clarification!